src/ophyd_async/sim/sim_pattern_generator.py

```diff
@@ -17,15 +22,16 @@ def __init__(
         config_sigs: Sequence[AsyncReadable] = [],
```

Comment:
Seeing unnoticed mutable defaults in the codebase during this review :hand_over_mouth:

```suggestion
        config_sigs: Sequence[AsyncReadable] = (),
```